code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -254,6 +254,8 @@
 /datum/reagent/water/holywater/on_mob_metabolize(mob/living/L)
 	..()
 	ADD_TRAIT(L, TRAIT_HOLY, type)
+	if(data)
+		data["misc"] = 0
```

Comment:
this is in on_mob_metabolize(), which I believe means that buckling yourself to and then unbuckling yourself from a stasis bed will reset your deconversion progress (unless self_consuming = TRUE ignores stasis?).

Reply:
shit you're right